samples/built-in-policy/app-service-audit-diagnostic-logs/azurepolicy.json

```diff
@@ -0,0 +1,57 @@
+{
+    "properties": {
+        "displayName": "Audit enabling of diagnostic logs in App Services",
+        "policyType": "BuiltIn",
```

Comment:
Please remove this from all the policies: 
`"policyType": "BuiltIn",`

Policies in the samples repo should look like custom policies.